src/oc_erchef/apps/oc_chef_authz/src/oc_chef_authz_org_creator.erl

```diff
@@ -87,6 +88,7 @@
            {add_acl,
             [mk_tl(container, ?CONTAINERS), mk_tl(group, [admins, clients, users]), {organization}],
             ?ALL_PERMS, [{group, admins}]},
+           {add_acl, [{group, public_key_read_access}], [read, update], [{group, admins}]},
```

Comment:
These changes look good to me. I prefer read, update over adding read, grant. There probably would be no harm in adding others to the read list, but I'm fine with where it is.